android/build.gradle

```diff
@@ -33,10 +33,32 @@ def getExtOrDefault(name) {
 def getExtOrIntegerDefault(name) {
   return rootProject.ext.has(name) ? rootProject.ext.get(name) : (project.properties['PagerView_' + name]).toInteger()
 }
+def agpVersion = com.android.Version.ANDROID_GRADLE_PLUGIN_VERSION.tokenize('.')[0].toInteger()
```

Comment:
Could you explain the responsibilities of this code?

Reply:
RN 0.71 depended on classpath("com.android.tools.build:gradle:7.4.1"). So i get major version of build:gradle  to add namespace and remove package in AndroidManifest.